src/elements/content-sidebar/SidebarNavButton.js

```diff
@@ -13,18 +13,20 @@ import './SidebarNavButton.scss';
 type Props = {
     children: React.Node,
     interactionTarget: string,
+    isOpen?: boolean,
     onNavigate?: (SyntheticEvent<>, NavigateOptions) => void,
     sidebarView: string,
     tooltip: React.Node,
 };
 
-const SidebarNavButton = ({ children, interactionTarget, onNavigate, sidebarView, tooltip }: Props) => {
+const SidebarNavButton = ({ children, interactionTarget, isOpen, onNavigate, sidebarView, tooltip }: Props) => {
     const sidebarPath = `/${sidebarView}`;
 
     return (
         <Route path={sidebarPath}>
             {({ match }) => {
-                const isToggle = !!match && match.isExact;
+                const isMatch = !!match && match.isExact;
+                const isActive = () => !!isOpen && isMatch;
```

Comment:
Does this need to be a function? Can we determine `isActive` during `render()` and let the `NavButton IsActive` prop be a boolean?

Reply:
The goal is to mimic the behavior of [React Router's NavLink component](https://reacttraining.com/react-router/web/api/NavLink/isactive-func), which expects a function.